contrib/google.golang.org/grpc/option.go

```diff
@@ -50,6 +51,7 @@ func defaults(cfg *config) {
 	cfg.nonErrorCodes = map[codes.Code]bool{codes.Canceled: true}
 	// cfg.analyticsRate = globalconfig.AnalyticsRate()
 	cfg.analyticsRate = math.NaN()
+	cfg.ignoredMethods = map[string]struct{}{}
```

Comment:
It should be fine to remove this line, right?

Reply:
yeah right